cli/clidisplay/resources.go

```diff
@@ -41,12 +41,12 @@ func WorkspaceTags(writer io.Writer, tags types.TagBlocks) hcl.Diagnostics {
 			refs := tag.References()
 			tableWriter.AppendRow(table.Row{k, "??", strings.Join(refs, "\n")})
 
-			//refs := tb.AllReferences()
-			//refsStr := make([]string, 0, len(refs))
-			//for _, ref := range refs {
+			// refs := tb.AllReferences()
+			// refsStr := make([]string, 0, len(refs))
+			// for _, ref := range refs {
 			//	refsStr = append(refsStr, ref.String())
 			//}
-			//tableWriter.AppendRow(table.Row{unknown, "???", strings.Join(refsStr, "\n")})
+			// tableWriter.AppendRow(table.Row{unknown, "???", strings.Join(refsStr, "\n")})
```

Comment:
should we just get rid of this commented code or do we really need to keep it around?

Reply:
I'll cut it